LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017 Human Cell Atlas Authors, https://humancellatlas.org
+Copyright (c) 2019 Human Cell Atlas Authors, https://humancellatlas.org
```

Comment:
Think you want a range or list here: `2017, 2018, 2019`

Reply:
Huh that does seem possible. I just assumed it should be current. Perhaps @TimothyTickle knows better than I do.

Reply:
@TimothyTickle do you happen to know the answer here?

Reply:
I don't know the answer but did some googling: https://softwareengineering.stackexchange.com/a/210491